adafruit_debug_i2c.py

```diff
@@ -40,7 +40,7 @@
 
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_Debug_I2C.git"
-
+from re import sub
```

Comment:
try to remove re, it isnt available on m0 builds

Reply:
fixed!